src/bun.js/bindings/ErrorCode.cpp

```diff
@@ -224,6 +225,20 @@ extern "C" JSC::EncodedJSValue Bun__ERR_INVALID_ARG_TYPE(JSC::JSGlobalObject* gl
     auto message = makeString("The \""_s, arg_name, "\" argument must be of type "_s, expected_type, ". Received "_s, actual_value);
     return JSValue::encode(createError(globalObject, ErrorCode::ERR_INVALID_ARG_TYPE, message));
 }
+extern "C" JSC::EncodedJSValue Bun__ERR_INVALID_ARG_TYPE_static(JSC::JSGlobalObject* globalObject, const ZigString* val_arg_name, const ZigString* val_expected_type, JSC::EncodedJSValue val_actual_value)
+{
+    JSC::VM& vm = globalObject->vm();
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    auto arg_name = std::span<const unsigned char>(val_arg_name->ptr, val_arg_name->len);
```

Comment:
ptr is a tagged pointer. It could be UTF-16. It could be UTF-8. It could be Latin1.

Use the helper for converting to a WTF::String or a WTF::StringView instead of this because it will crash if we send it any user-defined data

Reply:
unless we want static to mean "ascii only", in which case we should add an assertion